src/braket/jobs/_entry_point_template.py

```diff
@@ -1,46 +1,68 @@
 run_entry_point = """
 import cloudpickle
-from braket.jobs import save_job_result
+import os
+from braket.jobs import get_results_dir, save_job_result
 from braket.jobs_data import PersistedJobDataFormat
 
+
+# set working directory to results dir
+os.chdir(get_results_dir())
+
+# create symlinks to input data
+links = link_input()
+
 # load and run serialized entry point function
 recovered = cloudpickle.loads({serialized})
 def {function_name}():
     result = recovered()
     if result is not None:
         save_job_result(result, data_format=PersistedJobDataFormat.PICKLED_V4)
+    # clean_links(links)
```

Comment:
Should this be commented?

Reply:
Ah right, that was just me testing default behavior